app/controllers/projects_controller.rb

```diff
@@ -4,9 +4,15 @@ class ProjectsController < ApplicationController
 
   # GET /projects
   # GET /projects.json
+
   def index
-    @projects = Project.all
     @metric_names = ProjectMetrics.metric_names
+    @projects = Project.all
```

Comment:
we're making two database queries for projects - first grabbing all, and then grabbing an ordered version.  What we might prefer is to be able to work with scopes (see other comment below), and then something like this:

```rb
  method = :order_by_metric_name
  method = :order_by_project_name if params[:type] == 'project_name'
  @projects = Project.send(method)
```